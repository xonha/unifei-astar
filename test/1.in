5
0F0T0
0F000
0F000
000T0
000T0
