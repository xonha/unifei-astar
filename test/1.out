Tomou -40 de dano! (quanto mais negativo melhor)
